src/main/resources/messages/LibertyBundles.properties

```diff
@@ -104,15 +104,20 @@ gradle.test.report.does.not.exist=Gradle Test Report Does Not Exist
 test.summary=Test Summary
 
 # View unit test report action
-view.unit.test.report=view unit test report
+view.unit.test.report=View unit test report
 unit.test.report.does.not.exist=Unit Test Report Does Not Exist
 
+# separate test labels for Liberty tool window
+tool.window.maven.integration.test.text=View integration test report
+tool.window.maven.unit.test.text=View unit test report
+tool.window.gradle.test.text=View test report
+
 # Run & Debug Configuration
 liberty.run.config.title=Liberty Run Configuration
 liberty.debug.port.unresolved=Unable to {0} in debug mode: could not resolve debug port for {1}.
-attaching.debugger=Attaching Debugger to Liberty Server JVM...
+attaching.debugger=Attaching Debugger
```

Comment:
I kind of like that it says "to Liberty Server JVM...." so that users know the process is related to Liberty, rather than just a general attach debugger action. What do you think?